cirq/contrib/quantum_volume/quantum_volume.py

```diff
@@ -338,6 +437,12 @@ def calculate_quantum_volume(
         routing_attempts: The number of times to route each model circuit onto
             the device. Each attempt will be graded using an ideal simulator
             and the best one will be used.
+        add_readout_error_correction: If true, add some parity bits that willx
+            later be used to detect readout error. WARNING: This makes the
+            simulator run extremely slowly for any width/depth of 4 or more,
+            probably because it doubles the circuit size. In reality, the
```

Comment:
You can probably strike "probably" :) This is indeed the reason

Reply:
Haha good to know, thanks.